app/controllers/admin/paypal_preferences_controller.rb

```diff
@@ -48,7 +52,8 @@ def index
       minimum_commission: minimum_commission,
       commission_from_seller: tx_settings[:commission_from_seller],
       minimum_listing_price: Money.new(tx_settings[:minimum_price_cents], @current_community.default_currency),
-      minimum_transaction_fee: Money.new(tx_settings[:minimum_transaction_fee_cents], @current_community.default_currency)
+      minimum_transaction_fee: Money.new(tx_settings[:minimum_transaction_fee_cents], @current_community.default_currency),
```

Comment:
Never mind. I just realized that `currency == @current_community.default_currency`.